onnxscript/function_libs/torch_lib/ops/core.py

```diff
@@ -7584,7 +7587,33 @@ def aten_scatter_reduce(
         self = op.Reshape(self, neg_1)
         index = op.Reshape(index, neg_1)
         src = op.Reshape(src, neg_1)
+
+    if not include_self:
+        if onnx_reduce == "max":
+            value = onh.from_array(
+                np.array([np.finfo(src.dtype.numpy()).min], dtype=src.dtype.numpy())
```

Comment:
E.g. could you add a comment on why we needed to use np.finfo min/max for this reduction type, for future readers?